docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,240 @@
+.. _start_simple:
+
+Simple Example
+==============
+
+This is perhaps the most minimal Envoy configuration which can be used to validate basic plain HTTP
+proxying. It is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Quick Start
+-----------
+
+This Quick Start gets you started using only files in the Envoy repo. The sections below explain
+this configuration file and execution steps in detail.
+
+Copy both :repo:`configs/Dockerfile` and
+:repo:`configs/google_com_proxy.v2.yaml` to the same directory on your local
+disk. Then, build and run the Dockerfile, and test out Envoy by sending a
+request to port 10000::
+
+  $ docker build -t envoy-google-test:v1 .
+  $ docker run -d -p 10000:10000 envoy-google-test:v1
+  $ curl -v localhost:10000
+
+The Dockerfile will build a container containing the latest version of Envoy,
+and copy a basic Envoy configuration into the container. This basic
+configuration tells Envoy to route incoming requests to \*.google.com.
+
+
+Configuration Walkthrough
+-------------------------
+
+Envoy can be configured both statically and dynamically.
+In the static configuration model, a YAML configuration file is used, which is passed
+as an argument on the command line.
+Envoy also has an extensive set of APIs that can be used for dynamic configuration.
+In this section, we'll walk through a simple static configuration.
+
+Below is the `google_com_proxy.v2.yaml` file.
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9901 }
+    static_resources:
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+              stat_prefix: ingress_http
+              route_config:
+                name: local_route
+                virtual_hosts:
+                - name: local_service
+                  domains: ["*"]
+                  routes:
+                  - match: { prefix: "/" }
+                    route: { host_rewrite: www.google.com, cluster: service_google }
+              http_filters:
+              - name: envoy.router
+      clusters:
+      - name: service_google
+        connect_timeout: 0.25s
+        type: LOGICAL_DNS
+        # Comment out the following line to test on v6 networks
```

Comment:
I think this might no longer be required given the DNS fix that came in a few weeks ago. Can you verify that this examples works with auto and if so remove to simplify?

Reply:
Yes, I checked the example, and also added the fix to cds to indicate the assumption if it's not specified (it will be part of this PR).

Reply:
https://github.com/envoyproxy/envoy/pull/2844